web/src/lib/components/listen-box.tsx

```diff
@@ -130,38 +130,54 @@ export default class ListenBox extends Component<Props, State> {
   }
 
   render() {
-    return <div className={'listen-box' +
-                           (this.state.loaded ? ' loaded' : '') +
-                           (this.state.playing ? ' playing' : '')}>
-      <div className="sentence-box">
-        <b style={!this.props.vote ? 'display: none;' : ''}>
-          What we asked them to read:
-        </b>{this.props.sentence}
-      </div>
-      <div onClick={this.onPlay} class="play-box">
-        <b style={!this.props.vote ? 'display: none;' : ''}>What they said:</b>
-        <Icon type={this.state.playing ? 'pause': 'play'} />
-      </div>
-      <div style={this.props.vote ? 'display: none;' : ''} class="delete-box"
-           onClick={this.onDelete}>
-        <Icon type="x"/>
-      </div>
-      <div style={!this.props.vote ? 'display: none;' : ''} className={'vote-box ' + (this.state.played ? '' : 'disabled')}>
-        <a onClick={this.voteYes}>
-          <Icon type="check"/>Yes!</a>
-        <a onClick={this.voteNo}>
-          <Icon type="x"/>Nope.</a>
+    return (
+      <div
+        className={
+          'listen-box' +
+          (this.state.loaded ? ' loaded' : '') +
+          (this.state.playing ? ' playing' : '')
+        }>
+        <div className="sentence-box">
```

Comment:
Just a note: even though Prettier is set to use single quote, somehow at JSX they enforces double quote at element properties.

Reply:
yup, i actually appreciate this, since it looks more like html to me